benchmarks/graphic_pipeline/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
 
-project(benchmark)
+project(graphic_pipeline)
```

Comment:
nit: graphic -> graphics

Reply:
fixed, moved the sample to 'graphics_pipeline'